org.mixedrealitytoolkit.input/InteractionModes/ProximityEnteredEventArgs.cs

```diff
@@ -0,0 +1,29 @@
+using UnityEngine;
```

Comment:
Missing copyright header

Reply:
Added in https://github.com/ms-RistoRK/MixedRealityToolkit-Unity/commit/1b5f80d9957ebd2762a1f6752d656f656674b671 resolving this.